terraform/state-bucket/prod/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_version = "= 0.11.10"
+}
+
+provider "aws" {
+  region  = "ap-south-1"
+  profile = "ecs-workshop"
+  version = "= 1.43.0"
+}
+
+resource "aws_s3_bucket" "terraform_state_storage" {
+  bucket = "prod-ecs-workshop-terraform-state"
+  acl    = "private"
+  region = "ap-south-1"
+
+  tags {
+    Name        = "prod-ecs-workshop-terraform-state"
+    Environment = "prod"
+  }
+
+  lifecycle {
+    prevent_destroy = false
```

Comment:
Any reason this is mentioned. Isn't prevent_destroy false default?

Reply:
Intended for it to be true. Committed the fix